src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -1,14 +1,38 @@
 package org.usfirst.frc4904.robot;
 
+import org.usfirst.frc4904.robot.RobotMap.Port;
```

Comment:
Why are you importing a class from the same file?

Reply:
not sure. I think eclipse imported it automatically